ScoutSuite/providers/azure/resources/sql/transparent_data_encryptions.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this necessary? I think the python interpreter now uses utf-8 implicitly.

Reply:
It's the default encoding for python 3 (not 2, which is raw data). Seeing how the refactoring is removing support for 2.x, we can leave this out I guess.